README.markdown

```diff
@@ -38,9 +44,9 @@ bootstrapped:
 
     git clone git://github.com/rspec/rspec-dev.git
     cd rspec-dev
-    bundle install --binstubs
-    bin/rake setup
-    bin/rake # runs tests in every repository
+    bundle install --path=vendor/bundle
```

Comment:
Agreed.  But since the rest of the text recommends isolating gems, that's a way to do it that doesn't depend on "Gemsets" which AFAIK is a feature of RVM and perhaps some other Rbuy managers.  But not `uru` and maybe not `rbenv` either.